helm/examples/efs/README.md

```diff
@@ -51,45 +52,114 @@ efs-csi-controller-594c7f67c7-mncgd   3/3     Running   0          32h
 efs-csi-node-8g8fk                    3/3     Running   0          32h
 efs-csi-node-fp8vt                    3/3     Running   0          32h
 ```
+# Storage for HPCC cluster
+There are three available storage life cycles/methods for the HPCC cluster.
+## 1. Dynamic storage with values-auto-efs.yaml
+This method uses the storage class "aws-efs-auto" to dynamically create Persistent Volume Claims (PVC). The PVCs are deleted when the HPCC cluster is deleted, meaning that the storage is deleted when the HPCC cluster is deleted. The storage can't be reused between different HPCC clusters.<br/>
 
-### Install "aws-efs" Stroage Class
+Under the efs directory, run the following command to create the "aws-efs-auto" storage class
 ```console
 kubectl apply -f storageclass.yaml
 ```
 To verify
 ```code
 kubectl get sc
 NAME            PROVISIONER             RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
-aws-efs         efs.csi.aws.com         Delete          Immediate              false                  32h
+aws-efs-auto    efs.csi.aws.com         Delete          Immediate              false                  9s
 ```
-
-## Deploy HPCC Storage
-## Deploy HPCC cluster with values-auto-efs.yaml
-It will automically create Persistent Volume Claims (PVC) and delete them when the HPCC cluster is deleted:<br/>
-Under the helm directory:
+To deploy the HPCC cluster, navigate to the helm directory and run:
 ```console
-  helm install myhpcc ./hpcc --set global.image.version=<HPCC Platform version> -f examples/efs/values-auto-efs.yaml
+  helm install myhpcc ./hpcc --set global.image.version=latest -f examples/efs/values-auto-efs.yaml
 ```
-
-## Deploy HPCC cluster with values-retained-efs.yaml
-This will require deploying HPCC PVCs first. PVCs will persist after the HPCC cluster is deleted.<br/>
-Under the helm directory:
+To cleanup, run
 ```console
-  helm install awsstorage examples/efs/hpcc-efs
-  # To start HPCC cluster:
-  helm install myhpcc hpcc/ --set global.image.version=latest -f examples/efs/values-retained-efs.yaml
+  helm uninstall myhpcc
+```
+## 2. Static storage within Kubernetes with values-retained-efs.yaml
+In this method, storage lives on the Kubernetes cluster level. It uses the helm chart "hpcc-efs-dynamic-pv" to manually create PVCs. The PVs are dynamically generated. The PVCs will persist after the HPCC cluster is deleted, meaning that the storage can be reused across different HPCC clusters.<br/>
```

Comment:
The PVs are dynamically,  should it be PVCs ??

Reply:
No, the PVCs aren't dynamically created, in the hpcc-efs-dynamic-pv/templates/create-pvc.yaml, they are actually manually created.